### DISCUSSION TITLE
Partition have been drop, but  the data is not deleted

### DISCUSSION THREAD

**Discussion by Chenfx-git** [apache/doris#39555] [A - General / Q&A]
![image](https://github.com/user-attachments/assets/445e1330-4e28-4b61-bc6d-7f28a8bb6aac)
![image](https://github.com/user-attachments/assets/20d619a0-6ba3-4620-b21f-d7da93eddaf5)
Partition have been drop, but actually, the disk data is not deleted.
1、ALTER TABLE table SET ("dynamic_partition.start" = "-2");
disk data is not deleted.
2、ALTER TABLE table DROP PARTITION p20240817
disk data is not deleted.
3、ALTER TABLE table DROP PARTITION p20240818 force
After a while, data deleted.

It seem to drop by force, data can be deleted.
path_gc_check_interval_second = 900. I don't see any trash.
How to clear data to free up disk space？
Thanks！

**Comment by ixzc** [comment]
if you use `drop` command, doris will not delete data immediately. in this way, you can recover your data. so if you want delete data immediately, you need to add `force` after `drop` command.
you can refer this
https://doris.apache.org/docs/2.1/sql-manual/sql-statements/Data-Definition-Statements/Drop/DROP-TABLE/

**Reply by cheungfeifu** [reply]
是不是有24h的回收站♻️？
